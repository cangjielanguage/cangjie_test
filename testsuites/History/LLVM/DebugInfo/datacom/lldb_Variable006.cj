// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_Variable006_expected.txt ../basecase/008_debug_func_no_arg.cj
// EXEC: cjc -g 008_debug_func_no_arg.cj
// EXEC: python3 DatacomTest.py lldb_Variable006_expected.txt



