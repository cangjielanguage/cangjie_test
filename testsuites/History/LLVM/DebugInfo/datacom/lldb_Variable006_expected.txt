lldb,lldb 008_debug_func_no_arg,008_debug_func_no_arg
b,b 12,008_debug_func_no_arg.cj:12
r,r,-> 12
cj_plocals,cj_plocals,a = 10
finish,finish,-> 4
s,s,-> 5
cj_plocals,cj_plocals,Functioncall001 = 30